### ISSUE TITLE
[Spell] [Death Knight] Ebon Gargoyle does not behave as intended.

### ISSUE THREAD

**Issue by Tamaxofalgalon** [dalaranwow/dalaran-wow#5827]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Several Issues with the **Ebon Gargoyle** summoned by **Summon Gargoyle**.
They are probably linked as they all stem from "Ebon Gargoyle" not counting as a pet.

1. **Ebon Gargoyle** does not react to /petattack or similar commands but will attack any target in range at random.
2. **Ebon Gargoyle** will not start attacking if the player doesnt move after it is summoned.

**Expected behaviour**: Tell us what should happen instead

1. **Ebon Gargoyle** should respond to commands you give your Ghoul like /petattack and /petfollow

2. **Ebon Gargoyle** should start attacking the target it was summoned on right away without the player needing to move to "activate" its attack.

**Steps to reproduce the problem**:

1. Summon  **Ebon Gargoyle** on one taget and use /petfollow or /petattack on another target
2.  Summon  **Ebon Gargoyle** and stand completely still --> No Attacks

**Include proofs for this behaviour**

Proof for "Should follow pet commands" (Taken out of  an EJ Guide from 2010):
 https://imgur.com/a/MLeOSsc
Proof for "Not attacking if you stand still": Easily testable and obvious.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

Summon Gargoyle: 49206

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by mrnuggets** [comment]
The amount of times this problem has been posted, and the amount of times it has been labeled as 'fixed', can we stop with the fake news and actually get it fixed?

**Comment by Chenocide** [comment]
Fixes are probably pending on the PTR and as such, nothing else will be done until the impact of said fixes can be measured on the live realm.

**Comment by Tamaxofalgalon** [comment]
Any updates?

**Comment by Tamaxofalgalon** [comment]
Would love some commincation on this! Even if this is not priority... @Lothloryen

**Comment by Lothloryen** [comment]
Hello there,

This issue should be fixed after the next scheduled maintenance.

Thanks for the report.

**Comment by Tamaxofalgalon** [comment]
Did some tests today - fix comfirmed working 
What an excelent start into 2020, good shit =)
